Source/Particles/Collision/BinaryCollision/PhotonPhoton/LinearBreitWheelerCrossSection.H

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2021 Neil Zaim
```

Comment:
The name is optional, I think we don't really keep them up-to-date.
However, I would definitely put either no name or the right name:
```suggestion
/* Copyright 2023 Arianna Formenti
```